pkg/controller/stackconfigpolicy/controller.go

```diff
@@ -333,21 +374,107 @@ func (r *ReconcileStackConfigPolicy) doReconcile(ctx context.Context, policy pol
 		}
 
 		// update the ES resource status for this ES
-		status.UpdateResourceStatusPhase(esNsn, newResourceStatus(currentSettings, expectedVersion), configAndSecretMountsApplied)
+		err = status.UpdateResourceStatusPhase(esNsn, newElasticsearchResourceStatus(currentSettings, expectedVersion), configAndSecretMountsApplied, policyv1alpha1.ElasticsearchResourceType)
+		if err != nil {
+			return results.WithError(err), status
+		}
+	}
+
+	// Add dynamic watches on the additional secret mounts
+	// This will also remove dynamic watches for secrets that no longer are refrenced in the stackconfigpolicy
+	if err = r.addDynamicWatchesOnAdditionalSecretMounts(policy); err != nil {
+		return results.WithError(err), status
 	}
 
 	// reset/delete Settings secrets for resources no longer selected by this policy
-	results.WithError(handleOrphanSoftOwnedSecrets(ctx, r.Client, k8s.ExtractNamespacedName(&policy), configuredResources))
+	results.WithError(handleOrphanSoftOwnedSecrets(ctx, r.Client, k8s.ExtractNamespacedName(&policy), configuredResources, nil, policyv1alpha1.ElasticsearchResourceType))
 
-	// requeue if not ready
-	if status.Phase != policyv1alpha1.ReadyPhase {
-		results.WithResult(defaultRequeue)
+	return results, status
+}
+
+func (r *ReconcileStackConfigPolicy) reconcileKibanaResources(ctx context.Context, policy policyv1alpha1.StackConfigPolicy, status policyv1alpha1.StackConfigPolicyStatus) (*reconciler.Results, policyv1alpha1.StackConfigPolicyStatus) {
+	log := ulog.FromContext(ctx)
+	log.V(1).Info("Reconcile Kibana Resources", "namespace", policy.Namespace, "policy_name", policy.Name)
```

Comment:
Something is wrong with the values, some of them are duplicated like `namespace` or `policy_name` are duplicated:

```
2023-12-07T15:38:42.467+0100    DEBUG   stackconfigpolicy-controller    Reconcile Kibana Resources      {"service.version": "0.0.0-SNAPSHOT+00000000", "iteration": "11", "namespace": "elastic-system", "policy_name": "test-stack-config-policy", "namespace": "elastic-system", "policy_name": "test-stack-config-policy"}
```